pytensor/link/c/cmodule.py

```diff
@@ -2764,6 +2850,11 @@ def default_blas_ldflags():
 
                 # Canopy
         if "Canopy" in sys.prefix:
+            warnings.warn(
```

Comment:
I think we could actually remote those, I don't think they exist. Or just directly raise an Error.

Reply:
We should remove them, but I don’t we should do so without a warning that lasts for one minor version

Reply:
I'm pretty sure the % of users using `canopy` and `pytensor` is 0. From https://docs.enthought.com/canopy/index.html: "Canopy provides Python 2.7 and 3.5".